### PR TITLE
Warp dba

### PR DESCRIPTION
**Pull request by julbar**

### CODE REVIEW

**Review by shining-cat — APPROVED** [review]
looks good overall to me, but I guess the only way to be certain is to have as much people (designer and such) as possible paying around with the app in DBA flavour to ensure the result is fine.
From a structure point of view, LGTM

File: app/src/main/java/com/schibsted/nmp/warpapp/Theme.kt [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
 import androidx.compose.runtime.Composable
+import com.schibsted.nmp.warp.brands.dba.DbaWarpTheme
 import com.schibsted.nmp.warp.brands.finn.FinnWarpTheme
 import com.schibsted.nmp.warp.brands.tori.ToriWarpTheme
 import com.schibsted.nmp.warp.components.legacy.LegacyWarpTheme
 
 const val FINN = "finn"
 const val TORI = "tori"
+const val DBA = "dba"
```
Comment (martintoften) [inline app/src/main/java/com/schibsted/nmp/warpapp/Theme.kt:11]: An enum would be useful here!

File: warp-finn/src/main/res/values/finn_colors.xml [hunk 2]
```diff
@@ -65,10 +65,10 @@
 
     <color name="gray50">#F6F6F6</color>
     <color name="gray100">#F0F0F2</color>
-    <color name="gray200">#D1D1D6</color>
-    <color name="gray300">#BBBBC2</color>
-    <color name="gray400">#A5A5AD</color>
-    <color name="gray500">#80808A</color>
+    <color name="gray200">#DEDEE3</color>
```
Comment (martintoften) [inline warp-finn/src/main/res/values/finn_colors.xml:68]: New tori and finn colors?

File: warp-dba/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            )
+        }
+        debug {
+            isMinifyEnabled = false
+        }
+    }
+    testOptions.unitTests {
+        isIncludeAndroidResources = true
+    }
+    compileOptions {
```
Comment (martintoften) [inline warp-dba/build.gradle.kts:35]: Having a gradle script to add basic stuff like java version and compose could be useful to make it less verbose

**Review by martintoften — APPROVED** [review]
Left some minor comments but looks good 👍 I can add this to the nmp repo

File: warp-finn/src/main/res/values/finn_colors.xml (hunk 2 above)
Comment (julbar) [reply warp-finn/src/main/res/values/finn_colors.xml:68]: They have been updated a while back, i noticed that i forgot to change the values in xml. Compose was fine :)

**Pull request merged**